tests/test_common.py

```diff
@@ -1,9 +1,9 @@
-import unittest
 import tempfile
-import utils
-import os
+import unittest
 from os import path
-from common import *
+from recipemd.common import *
```

Comment:
The package does not need to be installed if "python3 -m unittest discover" or "pytest" are run from the top-level directory. This is why Travis still works. (Apparently, pytest does some magic and can also be run from the "tests/" directory while unittest has to be run from the top-level directory.)

Using ``os.path.abspath('..')`` like in the link you sent seems like a bad idea to me since if you run the unit tests from the top-level directory of the repo, it will add some directory outside of the repo to the path. So I would say something like ``sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))`` would be necessary for the approach with the ``context.py`` as described on python-guide.org.

I quickly tested this, but actually the relative imported suggested on the page does not work for me. I created an empty ``__init__.py``, a ``context.py``:
```
import sys
import os
sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

import recipemd
```
and changed the imports like this:
```
from .context.recipemd import utils
```

I think I followed what is written on that page, but this fails with "Parent module '' not loaded, cannot perform relative import". Do you have some idea how to fix this or do you even still want to follow that approach, knowing it now already works without installing?